kafka-client/src/main/java/dev/responsive/kafka/store/RemoteWriter.java

```diff
@@ -14,17 +14,19 @@
  * limitations under the License.
  */
 
-package dev.responsive.db;
```

Comment:
to me this actually feels like it fits better in the db package alongside the factory

Reply:
hah... weird effect of git, this file wasn't actually renamed I created a new one. Happy to move packages though.